lib/src/common/variables.ts

```diff
@@ -867,6 +873,16 @@ export const componentTokens = {
     scrollBarThumbColor: CoreTokens.color_grey_700,
     scrollBarTrackColor: CoreTokens.color_grey_300,
     sortIconColor: CoreTokens.color_white,
+    actionIconColor: CoreTokens.color_purple_700,
+    disabledActionIconColor: CoreTokens.color_grey_500,
+    hoverActionIconColor: CoreTokens.color_purple_700,
+    focusActionIconColor: CoreTokens.color_purple_700,
+    activeActionIconColor: CoreTokens.color_purple_700,
+    actionBackgroundColor: CoreTokens.color_transparent,
+    disabledActionBackgroundColor: CoreTokens.color_transparent,
+    hoverActionBackgroundColor: CoreTokens.color_grey_100,
+    focusActionBorderColor: CoreTokens.color_blue_600,
+    activeActionBackgroundColor: CoreTokens.color_grey_300,
```

Comment:
Adding new Component Tokens may require updates in advanced-theme / advanced-schema

Reply:
Yes that should be added in a different PR